securedrop/journalist.py

```diff
@@ -645,7 +649,7 @@ def download_single_submission(filesystem_id, fn):
 @app.route('/reply', methods=('POST',))
 @login_required
 def reply():
-    """Attempt to send a Reply from a Journalist to a Source. Empty
+    """Attempt to send a reply from a journalist to a source. Empty
```

Comment:
I was asked to revert "reply" and "source" back to "Reply" and "Source" in #2248

Reply:
Sorry about that, I thought I reverted the changes indeed but ... Good catch, thanks.